pkg/canmsgdefs/can_msg_defs.go

```diff
@@ -0,0 +1,68 @@
+package canmsgdefs
```

Comment:
Nit: Can you either create a `README` file in this directory that these files are autogenerated or make a note somewhere? And put a `TODO` to automate rolling the CAN message definitions/highlight that the process currently is a copy/paste.

Reply:
I've added this as a comment in this file. Should I also create a ticket to automate the process or do you think that would be excessive?

Reply:
I've also added this note to the Golang output in a [codegen](https://github.com/uw-midsun/codegen-tooling/pull/34) repo PR.

Reply:
Yeah, that's fine, just toss it into the backlog.